lib/Url.js

```diff
@@ -18,7 +22,7 @@ Url = function (opts) {
     };
   }
 
-  this._url = opts.url;
+  this._url = new URL(opts.url).href;
```

Comment:
I have a small concern that if you already have a URL in your database, it would never be upserted by `DbUrlList` (https://github.com/brendonboshell/supercrawler/blob/master/lib/DbUrlList.js#L196) so potentially would continuously attempt to re-crawl that URL?

Reply:
I think the `RedisUrlList` does something similar.

Reply:
@brendonboshell
Thanks for the review, and sorry for my delayed reply. I was considering how to solve the following issue you mentioned.

> if you already have a URL in your database, it would never be upserted by DbUrlList so potentially would continuously attempt to re-crawl that URL?

I still don't have an idea to solve this issue, so let me have time to consider that.